src/createServer.js

```diff
@@ -1,8 +1,105 @@
 'use strict';
 
+const url = require('url');
+const http = require('http');
+const fs = require('fs');
+const path = require('path');
+
 function createServer() {
-  /* Write your code here */
-  // Return instance of http.Server class
+  return http.createServer((req, res) => {
+    const normalizeUrl = new url.URL(req.url, `http://${req.headers.host}`);
+    const pathName = normalizeUrl.pathname;
+    const arrayFromPathName = pathName.slice(1).split('/');
+
+    if (pathName.includes('//')) {
+      res.writeHead(404, {
+        'Content-Type': 'text/plain',
+      });
+
+      res.end(
+        JSON.stringify({
+          error: 'Path to file should be like this /file/folderName/filename',
+        }),
+      );
```

Comment:
simply use strings in every res.end()
```suggestion
      res.end('Path to file should be like this /file/folderName/filename');
```